theme/less/_variables.less

```diff
@@ -1,4 +1,5 @@
 // Colors
-@primary: #728448;
+@primary: white;
```

Comment:
The alternative option is to have no background-color in header section. OK

Reply:
Somehow I did not managed to imply it in this commit but it was fixed by removing the `background-color` property.

Reply:
Fare enough, but we now have variable that is not used anywhere in the code. I suggest one of those:
- Remove variable completely.
- Use more meaning-full color value and reuse where applicable.
- Revert the color to it's initial state `#728448` to provide less noise in the review and take care of it in the future.

I lean towards options 1 and 2, because we will need to create a followup issue for 3 to keep track of what needs to be done.

Reply:
The issue of colors is not simple. we need to use less colors, in the other hand the colors of drupal site varies in close range and it is hard to choose which one is primary and secondary. I suggest to remove variables leaving the file empty and return to this issue when site and its colors will be more definite.

Reply:
Agree